src/__tests__/core/web_api/p2_api.test.js

```diff
@@ -79,6 +83,14 @@ describe('Auth0APIClient', () => {
         const client = getClient({ params: { state: 'foo' } });
         expect(client.authOpt.state).toBe('foo');
       });
+      it('should set nonce from options.nonce', () => {
+        const client = getClient({ nonce: 'foo' });
+        expect(client.authOpt.nonce).toBe('foo');
+      });
+      it('should set nonce from options.params.nonce', () => {
+        const client = getClient({ params: { nonce: 'foo' } });
+        expect(client.authOpt.nonce).toBe('foo');
+      });
```

Comment:
what if both `nonce` are set, which one should win? Add that test and document this behavior somewhere.

Reply:
added